### DISCUSSION TITLE
Does hmApp.alarmNew(option) survive reboots?

### DISCUSSION THREAD

**Discussion by jlcvp** [zepp-health/zeppos-forum#60] [Q&A] [answered]
Hey everyone, I'm writing an app that will rely on the calls for `hmApp.alarmNew(option)` to properly work and two thing grabbed my attention, and I couldn't find the answer in the docs:

1. Will this alarm survive a reboot of the device?
2. If yes, will the reference descriptor be the same and will cancel when calling `hmApp.alarmCancel(alarmId)`?

**Comment by htoooth** [comment, marked as answer]
@jlcvp Not supported at present. If a reboot happened, the alarm was not available.
